tests/expected/arbitrary/integers/expected

```diff
@@ -0,0 +1,222 @@
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:32:1 in function check_usize
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:32:1 in function check_usize
+
+Checking harness check_i128...
+
+Status: SATISFIED\
+Description: "This may be true"\
+main.rs:24:1 in function check_i128
+
+Status: SATISFIED\
+Description: "This may also be true"\
+main.rs:24:1 in function check_i128
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:24:1 in function check_i128
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:24:1 in function check_i128
+
+Checking harness check_i64...
+
+Status: SATISFIED\
+Description: "This may be true"\
+main.rs:23:1 in function check_i64
+
+Status: SATISFIED\
+Description: "This may also be true"\
+main.rs:23:1 in function check_i64
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:23:1 in function check_i64
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:23:1 in function check_i64
+
+Checking harness check_i8...
+
+Status: SATISFIED\
+Description: "This may be true"\
+main.rs:20:1 in function check_i8
+
+Status: SATISFIED\
+Description: "This may also be true"\
+main.rs:20:1 in function check_i8
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:20:1 in function check_i8
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:20:1 in function check_i8
+
+Checking harness check_isize...
+
+Status: SATISFIED\
+Description: "This may be true"\
+main.rs:25:1 in function check_isize
+
+Status: SATISFIED\
+Description: "This may also be true"\
+main.rs:25:1 in function check_isize
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:25:1 in function check_isize
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:25:1 in function check_isize
+
+Checking harness check_u16...
+
+Status: SATISFIED\
+Description: "This may be true"\
+main.rs:28:1 in function check_u16
+
+Status: SATISFIED\
+Description: "This may also be true"\
+main.rs:28:1 in function check_u16
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:28:1 in function check_u16
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:28:1 in function check_u16
+
+Checking harness check_i32...
+
+Status: SATISFIED\
+Description: "This may be true"\
+main.rs:22:1 in function check_i32
+
+Status: SATISFIED\
+Description: "This may also be true"\
+main.rs:22:1 in function check_i32
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:22:1 in function check_i32
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:22:1 in function check_i32
+
+Checking harness check_i16...
+
+Status: SATISFIED\
+Description: "This may be true"\
+main.rs:21:1 in function check_i16
+
+Status: SATISFIED\
+Description: "This may also be true"\
+main.rs:21:1 in function check_i16
+
+Status: SATISFIED\
+Description: "v1 may be MAX"\
+main.rs:21:1 in function check_i16
+
+Status: SATISFIED\
+Description: "v1 may be MIN"\
+main.rs:21:1 in function check_i16
```

Comment:
Please don't include line and column in `expected` files, unless they are extremely important. It can be annoying to update them. E.g.: Adding a comment in this file would require changing all the lines in the `expected` file.

Reply:
Sure, I'll remove them. What is important to include is the `in function <function-name>` on the next line to make sure this check passes in all harnesses. For example, if the `expected` file only had:
```
Status: SATISFIED\
Description: "v1 may be MIN"
```
then the test will pass if this check was satisfied in _any_ harness. We need to make sure it passes in _all_ harnesses.

Reply:
Done.